.travis.yml

```diff
@@ -1,15 +1,3 @@
 language: node_js
 node_js:
   - "6"
```

Comment:
Why aren't we dropping Travis altogether? Circle is faster, yet we'll keep waiting on Travis builds to finish.